File: tests/conformance/fixtures/go/err-duplicate.go
package main

import sykli "github.com/yairfalse/sykli/sdk/go"

func main() {
	p := sykli.New()

	p.Task("test").Run("go test ./...")
	p.Task("test").Run("go test -race ./...")

	p.Emit()
}

File: tests/conformance/fixtures/elixir/err-duplicate.exs
use Sykli

pipeline do
  task "test" do
    run "go test ./..."
  end
  task "test" do
    run "go test -race ./..."
  end
end
|> Sykli.Emitter.validate!()
|> Sykli.Emitter.to_json()
|> IO.puts()
